code/src/cljs/sixsq/nuvla/ui/deployments/utils.cljs

```diff
@@ -160,20 +160,22 @@
          label)])))
 
 (defn build-bulk-filter
-  [{:keys [::spec/select-all?
-           ::spec/selected-set
-           ::spec/additional-filter
-           ::spec/state-selector] :as db}]
-  (if select-all?
-    (get-filter-param
+  ([{:keys [::spec/select-all?
+            ::spec/selected-set] :as db}]
```

Comment:
Not working build-bulk-filter is returning empty string.
::spec/selected-set and ::spec/select-all? seems to be replaced by ::spec/select but not everywhere.
Events: ::events/select-all-page and ::events/select-all etc. not used. 

Something is not pushed?

Reply:
Forgot to remove the events, they are now part of the bulk selection feature.

Fixing other filter bug

Reply:
Should work now